### PR TITLE
Add get_direct_children_number command to keeper-client

### PR DESCRIPTION
**Pull request by xuzifu666**
### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Keeper client improvement: add get_direct_children_number command that returns number of direct children nodes under a 
 path

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

**Comment by xuzifu666** [comment]
@nikitamikhaylov  PTAL

**Comment by alexey-milovidov** [comment]
Squashing is needed.

**Comment by xuzifu666** [comment]
> Squashing is needed.

hi, @alexey-milovidov please tell me where is need to squashing if possible，I don't konw why CI is not running

**Comment by alexey-milovidov** [comment]
There are many commits with message "fix", let's squash them to previous.

**Comment by xuzifu666** [comment]
> There are many commits with message "fix", let's squash them to previous.

ok，thanks for your suggestion，had squash all the commit to single one commit @alexey-milovidov

**Comment by xuzifu666** [comment]
@alexey-milovidov could you have a review plz?

File: programs/keeper-client/Commands.cpp [hunk 1]
```diff
@@ -475,6 +475,29 @@ void FourLetterWordCommand::execute(const ASTKeeperQuery * query, KeeperClient *
...
+
+    return true;
+}
+
+void GetDirectChildrenNumberCommand::execute(const ASTKeeperQuery * query, KeeperClient * client) const
+{
+    auto path = client->getAbsolutePath(query->args[0].safeGet<String>());
+
+    std::queue<fs::path> queue;
+    queue.push(path);
```
Comment (antaljanosbenjamin) [inline programs/keeper-client/Commands.cpp:494]:
```suggestion
```

Comment (xuzifu666) [reply programs/keeper-client/Commands.cpp]: @antaljanosbenjamin done

**Comment by robot-ch-test-poll2** [comment]
*This is an automated comment for commit 8eec67b359802e71658031950b057f1028533d5f with description of existing statuses. It's updated for the latest CI running*

[❌ Click here](https://s3.amazonaws.com/clickhouse-test-reports/55898/8eec67b359802e71658031950b057f1028533d5f/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td><a href="https://github.com/ClickHouse/ClickHouse/actions/runs/6603318930">✅ success</a></td></tr>
<tbody>
</table>
</details>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/55898/8eec67b359802e71658031950b057f1028533d5f/ci_running.html">❌ failure</a></td></tr>
<tbody>
</table>

**Review by antaljanosbenjamin — APPROVED** [review]

**Comment by xuzifu666** [comment]
@antaljanosbenjamin ci seems well done

**Comment by antaljanosbenjamin** [comment]
Yes, the CI seems good. There is failure I would like to check it out. I did a bit of digging, and I assume it is not connected to your changes. I will go through your PR once more and probably merge it in a few hours.

**Comment by xuzifu666** [comment]
> Yes, the CI seems good. There is failure I would like to check it out. I did a bit of digging, and I assume it is not connected to your changes. I will go through your PR once more and probably merge it in a few hours.

ok，thanks for your review- @antaljanosbenjamin

**Pull request merged**